packages/market-service/src/index.ts

```diff
@@ -7,20 +8,30 @@ import {
   PriceHistoryType
 } from '@shapeshiftoss/types'
 import { FindAllMarketArgs } from '@shapeshiftoss/types/src'
+import { Yearn } from '@yfi/sdk'
 
 import { CoinGeckoMarketService } from './coingecko/coingecko'
+import { YearnMarketCapService } from './yearn/yearn'
+
+// YearnMarketCapService deps
+const provider = new JsonRpcProvider('https://dev-daemon.ethereum.shapeshift.com')
```

Comment:
can't hard code this - can we inject config into the market providers?

Reply:
switched to process.env